scss/_patterns_navigation.scss

```
@@ -192,6 +198,45 @@ $lightness-threshold: 70;
     }
   }
 
+  .p-navigation__tagged-logo {
+    margin-right: 0;
+
+    &-tag {
```

Please use full class names here to make it easier to find/read it later.
```suggestion
    .p-navigation__tagged-logo-tag {
```

---

Actually, the `p-navigation__tagged-logo-tag` seems overly long. Maybe we can shorten it to `p-navigation__logo-tag`?